### PR TITLE
Adding humio group CRD. 

### PR DESCRIPTION
**Pull request by CJNA**
Will add logic to parse these CRD.
Would like to get feedback before moving forward on reconciler logic.

### CODE REVIEW

**Comment by ryanfaircloth** [comment]
Is anyone continuing to work on this one?

**Comment by CJNA** [comment]
Hey Ryan.
I am look over at is now.
Sorry was pushed to do other stuffs

File: charts/humio-operator/crds/core.humio.com_humiogroups.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+              lookupName:
+                description: lookupName refers to an displayName of type HumioGroups. 
+                Reconciler will try to find a HumioGroup with given displayName, and if there is no such instance, it will try to create it
+                type: string
+              defaultQueryString:
+                description: DefaultQueryString refers to an defaultQueryString of type HumioGroups. 
+                If, Humio Group with given displayName is found, Reconciler will reconcile it to this defaultQueryString. If reconciler create new HumioGroup, 
+                HumioGroup will be set to have this defaultQueryString value as defaultQueryString
+                type: string
+              reconcileSearchDomainQueryPrefix:
```
[the remark this answers is not in the document]
Comment (CJNA) [reply charts/humio-operator/crds/core.humio.com_humiogroups.yaml:63]: Sorry I think I was confused. Will remove this var.

File: charts/humio-operator/crds/core.humio.com_humiogroups.yaml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                  DefaultRole refers to a role this HumioGroup should assume as default
+                type: object
+                properties:
+                  displayName:
+                    type: string
+                  viewPermissions:
+                    type: array
+                      items: 
+                        type: string
+                  systemPermissions:
```
[the remark this answers is not in the document]
Comment (CJNA) [reply charts/humio-operator/crds/core.humio.com_humiogroups.yaml:79]: I wrote something by myself, but will need Humio team to take a look a it.
Will ping them again

**Pull request closed without merging**